### ISSUE TITLE
Add `ReactNativeStyle` plugin in browser extension for react-native-web

### ISSUE THREAD

**Issue by jhen0409** [facebook/react-devtools#634]
#629 was removed the plugin in Chrome part, but it can be used for [`react-native-web`](https://github.com/necolas/react-native-web), that will be useful because the style editor in `Elements` tab of devtools is very complicated for RNW.

We can use the storybook of [`react-native-web`](https://github.com/necolas/react-native-web/tree/master/examples) to test this, just clone it and run `yarn` and `yarn examples`, and add the following code to [`.storybook/decorator-centered.js`](https://github.com/necolas/react-native-web/blob/master/examples/.storybook/decorator-centered.js):

```js
if (window.__REACT_DEVTOOLS_GLOBAL_HOOK__) {
  window.__REACT_DEVTOOLS_GLOBAL_HOOK__.resolveRNStyle = StyleSheet.flatten;
}
```

<img width="958" alt="2017-04-26 2 59 22" src="https://cloud.githubusercontent.com/assets/3001525/25402784/6b3f1aec-2a2c-11e7-901e-3529efdfc206.png">

It works with the browser extension v2.0.12. 

I can open a PR if this is accepted.

**Comment by gaearon** [comment]
Happy to take the PR!

**Comment by gaearon** [comment]
Fixed in #646.
